content/english/ml-machine-learning/01-regression.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "What is regression?"
+description: "Make computers learn to predict outcomes."
+prereq: "Python"
+icon: ""
+draft: false
+weight: 1
+---
+
+# What is regression?
+Regression is a technique to model the relationship between a feature (independent variables) and a variable feature (dependent variables). It helps us understand how the value of a dependent variable is changing based on the value of the independent variable. When applied properly it helps us predict values 😯!
+
+|![Regression](./resources/regression.png)|
+|:--:|
+|Regression|
+
+Regression gives a line or a curve by plotting dependent and independent variables. When we apply regression, we need to come up with a mathematical model that can explain the behavior we are seeing. If the mathematical model we use doesn't represent the data accordingly we need to find another model to use.
+
+## Example 1: Predicting the Weather Temperature
+For example, say you need to predict the temperature of a place. What do you need?
+
+- Altitude
+- Location
+- Month
+- Year
+- Humidity
+- Atmospheric Pressure
+
+The **temperature** is the dependent variable while the other properties we have mentioned above are the independent variables.
+
+## Fitting Your Model
+To fit a model is to make sure that the model you are using represents the information you have.
```

Comment:
Here you're talking about fitting a model, but in the exercise below, it's an exercise in feature selection. This is confusing. Fitting the model means optimizing the parameters of a given model (e.g. adjusting m and b in mx+b). It doesn't mean choosing what features to use for the prediction.